docs/user/installation.md

```diff
@@ -1 +1,63 @@
+
 # Installation
+
+SedTRAILS (Sediment TRAnsport vIsualization and Lagrangian Simulator). 
```

Comment:
Please read my comments about formatting in the previous file and apply them also here.